src/components/layout/Card.js

```diff
@@ -0,0 +1,57 @@
+import classnames from 'classnames';
+
+import { downcastRef } from '../../util/typing';
+
+/**
+ * @typedef {import('../../types').PresentationalProps} CommonProps
+ * @typedef {import('preact').JSX.HTMLAttributes<HTMLDivElement>} HTMLDivAttributes
```

Comment:
I'd suggest using `HTMLElement` here instead of `HTMLDivElement`. Same with other elements.